doc/glossary.rst

```diff
@@ -390,5 +390,11 @@
 	Teuthology
 		The collection of software that performs scripted tests on Ceph.
 
+        Zone
+                In the context of :term:`RGW`, a zone is a logical group that
+                consists of one or more :term:`RGW` instances. Zones can be
+                configured, and a zone's configuration state is stored in the 
```

Comment:
"Zones can be configured, and " seems superfluous.

Reply:
You're right. I'm still thinking in a tutorial register, and this is unambiguously reference material.